spigot/src/test/java/net/apartium/cocoabeans/spigot/listener/MyManager.java

```diff
@@ -0,0 +1,14 @@
+package net.apartium.cocoabeans.spigot.listener;
+
+public class MyManager {
+
+    private boolean hasBeenCreated = false;
```

Comment:
should probably be a counter? need to make sure it reaches 3

Reply:
I don't provide all of them with MyManager / plugin because I want to test that it know how to provide those objects

Reply:
I will create static counter but what's so bad about check all the register events from handler list?